### ISSUE TITLE
Backpropagation and Gradient Descent for Word Embedding

### ISSUE THREAD

**Issue by debadridtt** [keras-team/keras#10542]
Can anyone please tell me the significant difference, how does the weights of each word is updated during the training, using backpropagation and gradient descent in case of:

1) `model.add(Embedding(..))`
2) `from gensim.models import word2vec()` and then for  the  `model.add(Embedding(..))`
3) for `GloVe` embedding.
